.cspell/en-words.txt

```diff
@@ -34,6 +34,7 @@ discoverability
 dotnet
 Dyla
 dynatrace
+ebpf
```

Comment:
I added eBPF as a globally accepted word rather than just for this post, as it seems something general enough to be here.

Reply:
I am surprised that we don't have it already, because it is included in the textlintrc (as eBPF to ensure the correct writing)

Reply:
🤷 
I had the dictionary check complaining about it

Reply:
let's fix it then :-)

Reply:
Some background info / context ...

"ebpf" (all in lowercase) is _not_ proper spelling & capitalization, which is why there was no entry in the word list, and why there are rules to test and fix the capitalisation in the textlintrc file.

Usually when all lowercase spellings of mixed-case words occur in a page (such as a blog post) it is because it is used as part of an identifier, and in that case the exception should be added to the page's local dictionary.

Reply:
The only place where the term is used in all lowercase is URLs, or references to a repository name (where being in lowercase seems correct).